goflags.go

```diff
@@ -19,6 +19,11 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var (
+	EofYaml = errors.New("EOF")
```

Comment:
I think we should use `ErrEofYaml` (ref: https://go.dev/src/os/error.go)

Reply:
Done